dot/sync/tip_syncer_integeration_test.go

```diff
@@ -176,10 +173,8 @@ func TestTipSyncer_handleTick_case1(t *testing.T) {
 			targetNumber: fin.Number,
 			direction:    network.Descending,
 			requestData:  bootstrapRequestData,
-			pendingBlock: s.pendingBlocks.getBlock(common.Hash{0xb}),
```

Comment:
why is this removed? same with the others in this PR

Reply:
Honestly, I don't know why/how these got removed, probably during a rebase where I picked some wrong code to delete from conflict.  I've added these back in.